SIPS/sip-14.md

```diff
@@ -0,0 +1,81 @@
+---
+sip: 14
+title: Dynamic Permissions
+status: Draft
+discussions-to: 
+author: Frederik Bolding (@frederikbolding)
+created: 2023-10-19
+---
+
+## Abstract
+
+This SIP proposes changes to the snap manifest and new RPC methods that would allow snap developers to request additional permissions dynamically at runtime. This proposal outlines some of the details around this feature.

```

Comment:
```suggestion
This SIP proposes changes to the Snap manifest and new RPC methods that would allow Snap developers to request additional permissions dynamically at runtime. This proposal outlines some of the details around this feature.
```